src/swupd.h

```diff
@@ -388,6 +389,9 @@ extern void verify_set_picky_tree(char *picky_tree);
 /* repair.c */
 extern regex_t *compile_whitelist(const char *whitelist_pattern);
 
+/* mirror.c */
+extern bool mirror_is_set(void);
```

Comment:
mirror_is_set() can now be a static method.

Reply:
true